src/main/java/com/algorand/algosdk/v2/client/model/TransactionSignatureMultisigSubsignature.java

```diff
@@ -0,0 +1,32 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.util.Objects;
+
+import com.algorand.algosdk.v2.client.common.PathResponse;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TransactionSignatureMultisigSubsignature extends PathResponse {
+
+	/**
+	 * (pk) 
+	 */	@JsonProperty("public-key")
+	public String publicKey;
```

Comment:
Do we have a type for this? Also Digest?

Reply:
Drafted a solution. Should be tested.

Reply:
reverted back to byte[]. This can be done once the full list of Digest cases is identified, and the corresponding e2e test coverage cases are available.